tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -423,6 +423,12 @@ iface_namingmode/test_iface_namingmode.py::TestShowQueue:
     conditions:
       - "topo_name in ['m0', 'mx']"
 
+iface_namingmode/test_iface_namingmode.py::TestConfigInterface::test_config_interface_speed:
+  skip:
+    reason: "40G is not supported for cisco 8101 platform"
```

Comment:
1. The candidate port speed is from supported_speeds in state db. If can not support just 40G, please remove it from supported_speeds.
2. I also tried to change  the speed to 200G, the oper state will become down. Even I changed back to 100G, it is still down and can't be up.